### PR TITLE
Created a prototype app using the Google Maps Android API

### PR DESCRIPTION
**Pull request by felixlapierre**

### CODE REVIEW

File: app/src/release/res/values/google_maps_api.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<resources>
```
Comment (jun-loke) [inline app/src/release/res/values/google_maps_api.xml]: Should this file be here?

**Review by jun-loke — CHANGES_REQUESTED** [review]
Found a mis-named file

File: app/src/release/res/values/google_maps_api.xml (hunk 1 above)
Comment (felixlapierre) [reply app/src/release/res/values/google_maps_api.xml]: This one is in the release folder: i'll do the same thing as for the debug folder API key. You won't have to mess with it unless you're building in release.

Comment (jun-loke) [reply app/src/release/res/values/google_maps_api.xml]: sounds good

**Review by jun-loke — APPROVED** [review]
Runs on my machine. Looks good

**Comment by bogalla** [comment]
It works for me

**Review by bogalla — APPROVED** [review]

**Pull request merged**